### DISCUSSION TITLE
Is it possible to get the focused application id or name?

### DISCUSSION THREAD

**Discussion by seflerZ** [FreeRDP/FreeRDP#10742] [Q&A]
I wrote a RDP client and if I can get the focused window in RDP session, I can make some special shortcuts for it. Is it possible to get the current focused window name or id?

**Comment by akallabeth** [comment]
@seflerZ you mean desktop session or RAILS/Remote Application mode?
* in `desktop mode` the only way is to install a custom channel on the server (or do screen scraping)
* in rails mode you can get the window title as it is sent to the client

**Comment by seflerZ** [comment]
Desktop mode is better. However I have got the idea, thanks a lot.-------- 原始邮件 --------发件人： akallabeth ***@***.***>日期： 2024年10月16日周三 03:51收件人： FreeRDP/FreeRDP ***@***.***>抄送： Yao ***@***.***>, Mention ***@***.***>主    题： Re: [FreeRDP/FreeRDP] Is it possible to get the focused application id or name? (Discussion #10742)
@seflerZ you mean desktop session or RAILS/Remote Application mode?
in desktop mode the only way is to install a custom channel on the server (or do screen scraping)in rails mode you can get the window title as it is sent to the client
